ui/receive_page.go

```diff
@@ -0,0 +1,594 @@
+			walletAccounts:           make(map[int]walletAccount),
+			isWalletAccountModalOpen: false,
+		},
+	}
+
+	page.info.Inset, page.info.Size = layout.UniformInset(values.MarginPadding5), values.MarginPadding20
+	page.copy.Background = color.NRGBA{}
+	page.copy.Color = common.theme.Color.Primary
+	page.copy.Inset = layout.Inset{
+		Top:    values.SMarginPadding19p5,
+		Bottom: values.SMarginPadding19p5,
+		Left:   values.SMarginPadding16,
+		Right:  values.SMarginPadding16,
+	}
+	page.more.Color = common.theme.Color.IconColor
+	page.more.Inset = layout.UniformInset(values.MarginPadding0)
+	page.line.Color = common.theme.Color.Background
+	page.newAddr.Inset = layout.Inset{
+		Top:    values.MarginPadding20,
+		Bottom: values.MarginPadding20,
+		Left:   values.MarginPadding16,
+		Right:  values.MarginPadding16,
+	}
+	page.newAddr.Color = common.theme.Color.Text
+	page.newAddr.Background = common.theme.Color.Surface
+	page.newAddr.TextSize = values.TextSize16
+
+	return func(gtx C) D {
+		page.Handle(common)
+		page.wallAcctWidget.Handler(common)
+		return page.Layout(gtx, common)
+	}
+}
+
+func (pg *receivePage) Layout(gtx layout.Context, common pageCommon) layout.Dimensions {
+	pg.pageBackDrop(gtx)
+
+	pageContent := []func(gtx C) D{
+		func(gtx C) D {
+			return pg.pageSections(gtx, func(gtx C) D {
+				return pg.wallAcctWidget.accountSelectSection(gtx, common)
+			})
+		},
+		func(gtx C) D {
+			pg.line.Width, pg.line.Height = gtx.Constraints.Max.X, 1
+			pg.line.Color = common.theme.Color.Background
+			return pg.line.Layout(gtx)
+		},
+		func(gtx C) D {
+			return pg.pageSections(gtx, func(gtx C) D {
+				return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+					layout.Rigid(func(gtx C) D {
+						return pg.generateAddressSection(gtx, common)
+					}),
+					layout.Rigid(func(gtx C) D {
+						return pg.addressSection(gtx, common)
+					}),
+				)
+			})
+		},
+	}
+
+	dims := common.Layout(gtx, func(gtx C) D {
+		return common.UniformPadding(gtx, func(gtx C) D {
+			return layout.Flex{Axis: layout.Vertical}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					return layout.Inset{Bottom: values.MarginPadding16}.Layout(gtx, func(gtx C) D {
+						return pg.topNav(gtx, common)
+					})
+				}),
+				layout.Rigid(func(gtx C) D {
+					return common.theme.Card().Layout(gtx, func(gtx C) D {
+						return pg.pageContainer.Layout(gtx, len(pageContent), func(gtx C, i int) D {
+							return pageContent[i](gtx)
+						})
+					})
+				}),
+			)
+		})
+	})
+
+	if pg.wallAcctWidget.isWalletAccountModalOpen {
+		return common.Modal(gtx, dims, pg.wallAcctWidget.walletAccountSection(gtx, common))
+	}
+
+	return dims
+}
+
+func (pg *receivePage) pageSections(gtx layout.Context, body layout.Widget) layout.Dimensions {
+	return pg.theme.Card().Layout(gtx, func(gtx C) D {
+		gtx.Constraints.Min.X = gtx.Constraints.Max.X
+		return layout.UniformInset(values.MarginPadding16).Layout(gtx, body)
+	})
+}
+
+func (pg *receivePage) pageBackDrop(gtx layout.Context) {
```

Comment:
* backdrop is one word so, "pageBackdrop". 
* Kindly add a comment to this method describing what it does. There's a method named backdrop already, a comment will help to distinguish between the two at a glance.